### ISSUE TITLE
Add attention seeking animation to the NFT card

### ISSUE THREAD

**Issue by toiletgranny** [Joystream/atlas#2451]
## Scope

- [ ] Look into and try to implement a card "wiggle" (see below) upon opening the Settle auction page, whenever the NFT card is presented on the screen.

## Context

>@rafalpawlow I love this idea, brilliant! How about a subtle attention seeker [like this one](https://jsfiddle.net/toiletgranny/ym5Laujs/93/)? We could have it played only once, with... let's say a 2s delay, to make sure everything's properly loaded. The only question is, @eldiegod, are we going to be able to show all the lights, shines, and patterns in the animation like this one, even though the cursor might not be hovering over the NFT card?

@toiletgranny It's possible but it's not that easy because of what you mention, if we want this I think it's best to work on it in a separate ticket/PR

_Originally posted by @eldiegod in https://github.com/Joystream/atlas/issues/2412#issuecomment-1091533384_
